code/datums/elements/flavor_text.dm

```diff
@@ -5,12 +5,14 @@ GLOBAL_LIST_EMPTY(mobs_with_editable_flavor_text) //et tu, hacky code
 	id_arg_index = 3
 	var/flavor_name = "Flavor Text"
 	var/list/texts_by_atom = list()
-	var/addendum = "This can also be used for OOC notes and preferences!"
+	var/addendum = ""
 	var/always_show = FALSE
 	var/max_len = MAX_FLAVOR_LEN
 	var/can_edit = TRUE
+	/// For preference/DNA saving/loading. Null to prevent.
+	var/save_key
```

Comment:
Also add a new variable used to override the standard examine list input on lines on lines 64-68.
We need a simple `[OOC]` text href button, without the first 37 characters of possible kinks.